### ISSUE TITLE
Sounds please

### ISSUE THREAD

**Issue by Urox72** [budak7273/ArmorModules#27]
## Describe the bug

i´m using powersuit with allpak mod the only sound that is working is when powersuit reboots, no sound for overdraw no sound for thruster when using jetpak ohh the one sound that also work is the jump thuster module and switching between moduls
tried most variants wite

## To Reproduce
errr nah trying to GET sound 

## Expected behavior
sound plese? dying becaus i don hear when fuel is out
when i press thruster (space) i expect to get the appropriate sound

## Multiplayer?
nope single

## Other Mods
empty hands

What other mods do you have installed? Set your SMM to Show 'Enabled' mode, which lists all mods, including dependencies, and put a screenshot here. Alternatively, export and attach your SMM profile, or simply list what mods you have installed.

## Additional context

If you crashed, please attach your crash generated by the launcher to this report.
![](https://cdn.discordapp.com/attachments/555782140533407764/738104443849015387/unknown.png)
![smm](https://user-images.githubusercontent.com/86573541/148660926-a7e00705-dced-413a-bdbd-a674b8c0b6c8.JPG)

**Comment by budak7273** [comment]
Overdraw sound was fixed in PowerSuit 0.8.3, but it will be a while until I have the time to add flight sounds

**Comment by karlanke** [comment]
Are you open to PRs?

**Comment by budak7273** [comment]
@karlanke Yes, but this repo does not have the latest version of the source code for a number of reasons. If you are still interested in contributing, please contact me on discord via `Robb#6731`.
